internal/reports/aip_deletion.go

```diff
@@ -0,0 +1,137 @@
+package reports
```

Comment:
Can we move this reports package inside internal/storage? I'd like to keep the separation clear, and at some point do the same on ingest (moving specific ingest package inside internal/ingest, like internal/persistence), keeping the in the internal folder only those that are used by both services (like internal/event).